generator/default.p9k

```diff
@@ -546,11 +546,7 @@ function prompt_powerlevel9k_setup() {
           _P9K_RPROMPT_OVERRIDE=
           zle && zle .reset-prompt && zle -R
         }
-        autoload -Uz add-zle-hook-widget
-        zle -N _p9k_zle_line_finish
-        add-zle-hook-widget line-finish _p9k_zle_line_finish
```

Comment:
By the way, the reason for this change is that `add-zle-hook-widget` can lead to infinite loops if there is at least one plugin that wraps zle widgets the old school way instead of using `add-zle-hook-widget`. Until `add-zle-hook-widget` is fixed, it's must be avoided by everyone as a single user can break everything. The fix isn't difficult but the benefits are also low because wrapping widgets works fine. So the summary is that `add-zle-hook-widget` should be treated as if it doesn't exist and all users should be told to stop using it.

Reply:
Good to know. Thanks for the insights. 👍